geonode/people/migrations/0037_set_moderators_group_and_perms.py

```diff
@@ -0,0 +1,49 @@
+from django.db import migrations
+from django.db.migrations.operations import RunPython
+from django.contrib.auth.models import Group, Permission
+from django.contrib.contenttypes.models import ContentType
+from django.contrib.auth import get_user_model
+
+
+
+def create_group_and_assign_perms_forward_func(apps, schema_editor):
+    # create moderators group and assign permissions to it.
+    moderators_group, _ = Group.objects.get_or_create(name='moderators')
+    ctype = ContentType.objects.get_for_model(moderators_group)
+    approve_perm, _ = Permission.objects.get_or_create(
+        codename='approve_resourcebase',
```

Comment:
As from the original task, these will be global permissions, not resource specific, so they should be renamed to `approve_resources` and (below) `publish_resources`.
This will be in line with the `add_resources` permissions